abstractions/src/OpenSearch.Stack.ArtifactsApi/OpenSearchVersion.cs

```diff
@@ -87,43 +88,9 @@ public Artifact Artifact(Product product)
 		/// <summary>
 		///     Resolves an OpenSearch version using either format '$version' or '$ServerType-$version', where version could be 'x.y.z' or 'latest' or even 'latest-x'
 		/// </summary>
-		public static OpenSearchVersion From(string managedVersionString)
-		{
-			if (managedVersionString == null)
-				return null;
-
-			var serverType = ServerType.DEFAULT;
-			var hasServerType = Enum.GetNames(typeof(ServerType)).Any(s => managedVersionString.StartsWith(s, StringComparison.InvariantCultureIgnoreCase));
-			if (hasServerType)
-			{
-				var parts = managedVersionString.Split('-');
-				serverType = (ServerType)Enum.Parse(typeof(ServerType), parts[0], true);
-				managedVersionString = string.Join("-", parts.Skip(1));
-			}
-
-			if (managedVersionString.StartsWith("latest") && serverType == ServerType.OpenDistro)
-				// No releases for OpenDistro after 1.13.3 - all deveploment afterwards goes to OpenSearch
-				managedVersionString = "1.13.3";
-			if (managedVersionString.StartsWith("latest") && serverType == ServerType.ElasticSearch)
-				// OpenDistro (1.13.x - another not supported) and OpenSearch both based on Elasticsearch v.7.10.2, other versions are incompatible
-				managedVersionString = "7.10.2";
-
-			// TODO resolve `latest` and `latest-x` for OpenSearch
-
-			return new OpenSearchVersion(managedVersionString, ArtifactBuildState.Released, "");
-		}
-
-		internal static bool TryParseBuildCandidate(string passedVersion, out string version, out string gitHash)
-		{
-			version = null;
-			gitHash = null;
-			var tokens = passedVersion.Split(':');
-			if (tokens.Length < 2)
-				return false;
-			version = tokens[1].Trim();
-			gitHash = tokens[0].Trim();
-			return true;
-		}
+		public static OpenSearchVersion From(string managedVersionString) =>
+			// TODO resolve `latest` and `latest-x` for OpenSearchÏ
```

Comment:
```suggestion
			// TODO resolve `latest` and `latest-x` for OpenSearch
```
Somebody could get in trouble with unicode